pages/glossary/display-size.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Display size
+key: display-size
+unambiguous: true
+objective: true
+---
+
+The display size is the width and height at which a page is rendered. The display size is equal to the [clientWidth][] and [clientHeight][] of the [document element][] of the [top-level browsing context][].
+
+**Note**: The display size is not to be confused with the "resolution" of the operating system. Often a browser will be a single window in the operating system, with a width and height different from the resolution of the operating system. Often browsers also include additional user interface components, such as a URL bar, tab bar, and a bookmarks bar. None of these are included in the display size. In full screen mode the display size might be the same as the resolution of the operating system.
+
+**Note**: The display size includes, if rendered, all scrollbars.
```

Comment:
It however doesn't include the viewport scrollbars as it's defined in terms of `clientWidth` and `clientHeight` which exclude the size of scrollbars.